### ISSUE TITLE
About the results on SST 

### ISSUE THREAD

**Issue by Chanrom** [jihunchoi/unsupervised-treelstm#3]
Thanks for sharing the codes!
But I have some questions. I find that you calculate accuracy using:
`valid_accuracy = valid_accuracy_sum / num_valid_batches`
 in sst/train.py. I think the right way is:
`valid_accuracy = valid_accuracy_sum_all_samples / num_valid_samples`. 
They may not be equal.

Beside that, I manage to obtain comparable results in SST-2 (90.4 on test set), that is great. But only 50.7 on SST-5, is there anything I miss or I should try?

**Comment by jihunchoi** [comment]
Hi! Thanks for pointing out the mistake.
Actually at the time of implementation I was a bit lazy and just let that incorrect. But you can find that the evaluation script does not have the problem.
And I want to note that SST dataset is kind of small (even if we use all subtrees and sublabels in training), thus the result may fluctuate by tiny change in hyperparameters or initialization.
The best model I have (53.7 on test set) is trained under the following setting:
word dim 300, hidden dim 300, classifier dim 1024, dropout 0.5, adadelta, 840B 300d GloVe vectors with fine-tuning, lower-cased words, halving the learning rate when there is no improvement for 2 epochs, checkpointing on every 0.05 epoch.
